src/templates/UserList/UserListToolbar.tsx

```diff
@@ -47,7 +48,7 @@ export function UserListToolbar(props: UserListToolbarProps) {
             <></>
           )}
           {selectedRows.every((row) => {
-            return row.userType !== 'Admin';
+            return row.roles?.includes({ id: '1', name: 'user' });
```

Comment:
same story

Reply:
fixed in [ac83857](https://github.com/dorm-social-room-manager/dsrm-frontend/pull/38/commits/ac838579c3e681b31ed846e2ad54f55fc2a734e6)